requirements.txt

```diff
@@ -0,0 +1,2 @@
+discord.py
+flask
```

Comment:
In general, it's good to hardcode the versions in `requirements.txt` so our behavior doesn't change when rebuilding. The only truly safe thing is building a lockfile (maybe with `Poetry`) but we'll think about that later.